tsl/src/compression/compression.c

```diff
@@ -530,6 +532,50 @@ static SegmentInfo *segment_info_new(Form_pg_attribute column_attr);
 static void segment_info_update(SegmentInfo *segment_info, Datum val, bool is_null);
 static bool segment_info_datum_is_in_group(SegmentInfo *segment_info, Datum datum, bool is_null);
 
+static int32
+get_max_sequence_number(Relation compressed_table, int16 seq_num_column_num)
```

Comment:
the sequence number is not global but local per segmentby group so any function assuming it is global is not handling sequence number correctly